src/ui_ng/src/app/config/auth/config-auth.component.html

```diff
@@ -15,6 +15,30 @@
             </a>
         </div>
     </section>
+
+    <section class="form-block" *ngIf="showUAA">
+        <div class="form-group">
+            <label for="uaa-endpoint" class="required">{{'CONFIG.UAA.ENDPOINT' | translate}}</label>
+            <input type="text" id="uaa-endpoint" name="uaa-endpoint" placeholder="endpoint" size="35"
+            [(ngModel)]="currentConfig.uaa_endpoint.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
+        </div>
+        <div class="form-group">
+            <label for="uaa-id" class="required">{{'CONFIG.UAA.CLIENT_ID' | translate}}</label>
+            <input type="text" id="uaa-cid" name="uaa-client-id" size="35"
+            [(ngModel)]="currentConfig.uaa_client_id.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
```

Comment:
If setting placeholder, set for all for consistence

Reply:
I removed placeholders. UAA config has default value. And this input filed can not edit, placeholder may confuse the users.

Reply:
One minor comment: better to add a description for the 'uaa-id' field that the UAA CLIENT ID must have the following privilege: 

authorized-grant-types: password
scope: openid

@reasonerjt please correct me if anything missed.